Library/Formula/fcitx-remote-for-osx.rb

```diff
@@ -0,0 +1,18 @@
+class FcitxRemoteForOsx < Formula
+  desc "handle input method in command-line"
+  homepage "https://github.com/CodeFalling/fcitx-remote-for-osx"
+  url "https://github.com/CodeFalling/fcitx-remote-for-osx/archive/0.0.2.tar.gz"
+  sha256 "746505694090d98d5ad572758e7597c49d3bea1b00208a8272c3648aad15ad69"
+  option "with-input-method=", "Select input method: baidu-pinyin(default), baidu-wubi, sogou-pinyin, qq-wubi, squirrel-rime, osx-pinyin"
```

Comment:
Please leave a line between `sha256` and the first option. I'd also rather not have an open ended option, but do something like:

```ruby
option "with-baidu-pinyin", "Build with baidu-pinyin input method (default)"
option "with-baidu-wubi",  "Build with baidu-wubi input method"

def install
  args = []
  if build.with? "baidu-wubi"
    args << "with-input-method=baidu-wubi"
  elsif build.with? "something else"
    args << "with-input-method=example"
  else
    args << "with-input-method=baidu-pinyin"
  end

  system "./build.py", "build", *args
  bin.install "fcitx-remote-#{args}"
end
```

And so on. I saw @mikemcquaid comment on this setup earlier though, so if he's happy as-is, I won't fight about it.